### DISCUSSION TITLE
Indexing comments

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#902] [Q&A] [answered]
Continuing issue #900 

@akellbl4 And you can somehow check the indexing. I just tried to find pages in Google by comments from the site with the search flag "site:" and could not.

**Comment by paskal** [comment]
Could you please post the search query?

**Comment by akellbl4** [comment]
Indexation could be checked in Google Search Console. If comments were rendered they should be indexed.
I tried to search comments on one of the sites that is using remark and it works fine.

**Comment by uPagge** [comment, marked as answer]
@akellbl4 but can you tell me how to get to this section in the google console to see, something I did not find anything like that

@paskal oh, it seems that this particular page did not have time to get into the index, everything works with one of the older ones. Your project has just become even more wonderful for me, thanks for your work))
